fintrack/accounts/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+from banks.models import Branch
+from choices import AccountCategoryChoices
+from users.models import User
+
+
+class Account(models.Model):
+   
+
+    holder_name = models.CharField(max_length=100)
+    acc_number = models.CharField(max_length=20, primary_key=True)
```

Comment:
I want you to read about banks terminologies and use those here like title, number etc